fd.hcl

```diff
@@ -3,12 +3,22 @@ binaries = ["fd"]
 strip = 1
 test = "fd --version"
 
-platform "linux" {
-  source = "https://github.com/sharkdp/fd/releases/download/v${version}/fd-v${version}-x86_64-unknown-linux-musl.tar.gz"
+darwin {
+  vars = {
+    "osplus": "apple-darwin",
+  }
 }
 
-platform "darwin" {
-  source = "https://github.com/sharkdp/fd/releases/download/v${version}/fd-v${version}-x86_64-apple-darwin.tar.gz"
+linux {
+  vars = {
+    "osplus": "unknown-linux-musl",
+  }
+}
+
+source = "https://github.com/sharkdp/fd/releases/download/v${version}/fd-v${version}-${xarch}-${osplus}.tar.gz"
+
+version "8.3.2" "8.6.0" "8.7.0" "8.7.1" "9.0.0" "10.0.0" {
+  source = "https://github.com/sharkdp/fd/releases/download/v${version}/fd-v${version}-x86_64-${osplus}.tar.gz"
 }
 
 version "8.3.2" "8.6.0" "8.7.0" "8.7.1" "9.0.0" "10.0.0" "10.1.0" "10.2.0" {
```

Comment:
These old versions should be removed from this block.

Reply:
Done